examples/pytorch/rgcn/README.md

```diff
@@ -28,23 +28,22 @@ For mini-batch training, run with the following (available datasets are the same
 ```bash
 python3 entity_sample.py --dataset aifb
 ```
-For multi-gpu training (with sampling), run with the following (same datasets and GPU IDs separated by comma) 
+For multi-gpu training (with sampling), run with the following (same datasets and GPU IDs separated by comma)
 ```bash
 python3 entity_sample_multi_gpu.py --dataset aifb --gpu 0,1
 ```
 
-
 ### Link Prediction
-FB15k-237 in RAW-MRR
-```
-python link.py --gpu 0 --eval-protocol raw
-```
-FB15k-237 in Filtered-MRR
-```
-python link.py --gpu 0 --eval-protocol filtered
+
+Run with the following for link prediction on dataset FB15k-237 with filtered-MRR
+
+```bash
+python link.py
 ```
+> **_NOTE:_** By default, we use uniform edge sampling instead of neighbor-based edge sampling as in [author's code](https://github.com/MichSchli/RelationPrediction). In practice, we find it achieves similar MRR.
```

Comment:
```suggestion
> **_NOTE:_** By default, we use uniform edge sampling instead of neighbor-based edge sampling as in [author's code](https://github.com/MichSchli/RelationPrediction). In practice, we find it achieve similar MRR.
```

Reply:
I think you meant `it can achieve`?  I updated.

Reply:
Perhaps it's more safe to say `find that it can achieve`